buildscripts/bodo/pip/macos/patch_libs_for_pip.py

```diff
@@ -34,8 +34,10 @@ def patch_lib(fpath):
     #     @rpath/libarrow.500.dylib (compatibility version 500.0.0, current version 500.0.0)
     #     @rpath/libc++.1.dylib (compatibility version 1.0.0, current version 1.0.0)
     #     /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.0.0)
+    print("Loaded Libs for", fpath)
```

Comment:
do we want to keep prints in this file?

Reply:
Yes for sure, it was useful for debugging

Reply:
And it doesn't cause any issues